dbld/images/helpers/packages.manifest

```diff
@@ -202,18 +202,18 @@ gnupg                           [centos, fedora, debian, ubuntu]
 
 
 # packages to run Kira
-faketime                        [debian, ubuntu]
```

Comment:
Won't these removals have unexpected impacts on the images listed? In an other way: are we sure that these dependencies are only used in Kira and nothing else?

Reply:
In the past we have added these packages only for Kira purposes. 
For now I did not know that are they required somewhere else.
It will turns out now :)

Reply:
Aaaand it did. :)

lsb-release is moved to kira-only, while it was used for `devshell` image build.
So we cannot build `devshell` for sure.
```diff
-lsb-release                     [debian, ubuntu]                                                                                                                                                                   
+lsb-release                     [kira]                                                                                                                                                                             
```

I'll check around for other issues and send a PR soon.